### PR TITLE
dex,client: upgrade harness wallets and versions for spark hardfork

### PR DESCRIPTION
**Pull request by dev-warrior777**
## Firo will hard fork block 819300 for Lelantus Spark.

[Firo v0.14.13.1 Lelantus Spark](https://github.com/firoorg/firo/releases/tag/v0.14.13.1) release

### CODE REVIEW

**Comment by dev-warrior777** [comment]
`client/asset/firo/regnet_test.go` passes over multiple itertions
__simnet-trade-tests__ seem fine over multiple iterations

**Comment by dev-warrior777** [comment]
`client/asset/firo/regnet_test.go` passes over multiple iterations
__simnet-trade-tests__ seem fine over multiple iterations

File: dex/testing/firo/harness.sh [hunk 1]
```diff
@@ -258,23 +260,23 @@ ${CLI} ${GAMMA_CLI_CFG} "\$@"
...
+# sleep 1
+# ./mine-alpha 1
+# echo "Mining 3 blocks on beta"
+# ./mine-beta 3
+# sleep 2
+# echo "Reconnecting beta to alpha"
+# ./beta addnode 127.0.0.1:${ALPHA_LISTEN_PORT} onetry
+# sleep 2
+# EOF
+# chmod +x "./reorg"
```
Comment (buck54321) [inline dex/testing/firo/harness.sh:279]: What's up with this?

File: dex/testing/firo/harness.sh [hunk 2]
```diff
@@ -335,9 +337,9 @@ if [ -z "$NOMINER" ] ; then
   tmux send-keys -t $SESSION:5 "watch -n 15 ./mine-alpha 1" C-m
 fi
 
-# Live stop/start
-#tmux send-keys -t $SESSION:5 C-c C-m
-#tmux send-keys -t $SESSION:5 "watch -n 15 ./mine-alpha 1" C-m
+# Live stop/start - experimental
+# tmux send-keys -t $SESSION:5 C-c C-m
+# tmux send-keys -t $SESSION:5 "watch -n 15 ./mine-alpha 1" C-m
```
Comment (buck54321) [inline dex/testing/firo/harness.sh:342]: What's the purpose of these changes?

Comment (dev-warrior777) [reply dex/testing/firo/harness.sh:342]: This one is easier turn off and on mining .. it has been there a while but I can remove

File: dex/testing/firo/harness.sh (hunk 1 above)
Comment (dev-warrior777) [reply dex/testing/firo/harness.sh:279]: It's fine!
I had the script apart and 'all over the floor' while trying to figure out what had changed to make some rpcs not work. Then when putting stuff back together I did not incude this as I never used it. I will put it back in.

**Review by buck54321 — APPROVED** [review]
